hussain-shariff.json

```
@@ -0,0 +1,46 @@
+{
+  "name": "Hussain Shariff",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Mechanical engineer exploring web dev | Starting with Open Source journey",
+  "tags": ["reactJs", "nextJs", "tailwindCSS", "Sanity"],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/hussainshariff_" },
+    { "icon": "FaGithub", "url": "https://github.com/hussain-shariff"},
```

I believe there is a trailing comma here.
```suggestion
    { "icon": "FaGithub", "url": "https://github.com/hussain-shariff"}
```

---

Good spot @Njong392 👍

---

@Njong392 how do I make changes now?
Should I create a new json file and create PR or there is some way I can edit this?

---

> @Njong392 how do I make changes now? Should I create a new json file and create PR or there is some way I can edit this?

Nevermind I made the changes.
Is it okay now?